.travis.yml

```diff
@@ -17,7 +17,7 @@ before_script:
 script:
   - npm run lint
   - npm run test:js
-  - npm run build:production
+  - if [ "$TRAVIS_NODE_VERSION" == "6" ]; then echo "Skipping build on Node 6."; else make website; fi
```

Comment:
Instead of blacklisting Node 6, what about whitelisting just the one Node version you want to perform the site build on?

Even better, this could be switched to CircleCI and you could use a separate build job for the site, with whichever Node.js version you like.

I guess this is just a temporary transition state though, right?

Reply:
It seems safer to blacklist Node 6, because if we upgrade beyond Node 8 this would just stop running. Since I added `check-node-version` I could replace this with a semver expression, which expresses the intent even better.

Yea, it's just temporary until we get Node 8 on the server.